### PR TITLE
Toastr

### PR DESCRIPTION
**Pull request by RicardoNeves**

### CODE REVIEW

File: src/app/toast/_toast.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  .keds-toast {
+    left: 50%;
+    padding-bottom: 16px;
+    position: fixed;
+    top: 100%;
+  }
+
+  .keds-toast__container-wrapper {
+    background-color: keds-color-get($theme, core, shading, 90);
```
Comment (jmagalhaes-xing) [inline src/app/toast/_toast.scss]: Move to variable?

File: src/app/toast/_toast.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  $box-shadow-color: rgba(keds-color-get($theme, core, shading, 90), .15);
+
+  .keds-toast {
+    left: 50%;
+    padding-bottom: 16px;
+    position: fixed;
+    top: 100%;
+  }
+
+  .keds-toast__container-wrapper {
```
Comment (jmagalhaes-xing) [inline src/app/toast/_toast.scss]: Is it a container or a wrapper?

File: src/app/toast/components/toast/toast.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,117 @@
+import { Component, TemplateRef, ViewChild } from '@angular/core';
+import { ComponentFixture, fakeAsync, TestBed } from '@angular/core/testing';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { configureTestSuite } from 'ng-bullet';
+
+import { KEDS_TOAST_CONFIG, Toast } from '../../typings';
+
```
Comment (jmagalhaes-xing) [inline src/app/toast/components/toast/toast.component.spec.ts]: Remove extra line

File: src/app/toast/components/toast/toast.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    <ng-template #templateC>Template C</ng-template>
+    <keds-toast></keds-toast>
+  `,
+})
+class TestToastComponentComponent {
+
+  @ViewChild('templateA') templateA: TemplateRef<any>;
+  @ViewChild('templateB') templateB: TemplateRef<any>;
+  @ViewChild('templateC') templateC: TemplateRef<any>;
+  @ViewChild(ToastComponent) toast: ToastComponent;
```
Comment (jmagalhaes-xing) [inline src/app/toast/components/toast/toast.component.spec.ts]: Add line breaks in between?

File: src/app/toast/test.stories.ts [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+storiesOf('Toast Module/Toast Component', module)
+  .addDecorator(moduleMetadata({
+    declarations: [ToastStoryComponent],
+    imports: [
+      TextModule,
+      ToastModule,
+    ],
+  }))
+  .add('Default', () => ({
+    template: `<keds-toast-story></keds-toast-story>`,
```
Comment (jmagalhaes-xing) [inline src/app/toast/test.stories.ts]: You can add actions to test the component showing the toast

File: src/app/toast/_toast.scss (hunk 2 above)
Comment (RicardoNeves) [reply src/app/toast/_toast.scss]: it's a div that wraps the container.

**Pull request merged**